src/Symfony/Component/HttpFoundation/Session/Storage/MetadataBag.php

```diff
@@ -43,15 +36,10 @@ class MetadataBag implements SessionBagInterface
 
     /**
      * Unix timestamp.
```

Comment:
drop?

Reply:
I would keep it, to known what the int is.